itk-module.cmake

```diff
@@ -16,8 +16,10 @@ itk_module(ThinShellDemons
   COMPILE_DEPENDS
     ITKCommon
 	  ITKMesh
+    ITKIOMeshBase
```

Comment:
There is a mixture of tabs and spaces in this file. I think that using spaces exclusively is preferred.